WuV/onto_general.tex

```diff
@@ -71,6 +71,16 @@ \section{General Principles}\label{sec:onto:principles}
  This part is commonly called the \textbf{ABox} (A for assertional).
 \end{compactitem}
 
+Some sentences in following paragraph partially copied from \url{https://en.wikipedia.org/w/index.php?title=Abox&oldid=892872890}. License: \url{https://en.wikipedia.org/w/index.php?title=Special:CiteThisPage&page=Abox&id=892872890&wpFormIdentifier=titleform}.

```

Comment:
Here, the initial sentences are copied. Check with the URI which sentences precisely are copied. The rest of the paragraph was written by me.